### PR TITLE
Valentina

### PR DESCRIPTION
**Pull request by ValentinaBrigantina**

### CODE REVIEW

File: assets/styles/components/secondscreen.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.tt-second-fourth-screen__article-info {
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
```
Comment (suicaed) [inline assets/styles/components/secondscreen.css:4]: We don't need any additional space around section.

File: assets/styles/components/secondscreen.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    flex-direction: column;
+    justify-content: space-between;
+}
+.article-info__title {
+    font-size: 36px;
+    line-height: 65px;
+    font-family: 'Playfair Display', serif;
+}
+.article-info__title > h2 {
+    margin: 0;
```
Comment (suicaed) [inline assets/styles/components/secondscreen.css:24]: Disable the margin in reset.css

Comment (suicaed) [reply assets/styles/components/secondscreen.css:24]: Describe <h2> styles in common.css
Use rem in font-size.

File: assets/styles/components/secondscreen.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+}
+.article-info__text {
+    font-size: 17px;
```
Comment (suicaed) [inline assets/styles/components/secondscreen.css:27]: Use rem instead of px.

File: assets/styles/components/secondscreen.css [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    line-height: 30px;
```
Comment (suicaed) [inline assets/styles/components/secondscreen.css:28]: Rem.

File: index.html [hunk 5]
```diff
@@ -41,9 +41,44 @@
...
     <section class="tt-section tt-firstscreen tt-firstscreen__wrapper"></section>
+    
+    <section class="tt-section tt-second-fourth-screen tt-cecondscreen__wrapper">
+      <div class="tt-second-fourth-screen__article-info">
+        <img class="tt-second-fourth-screen__img" src="./assets/img/Thousand-02.png" alt="traveler">
+        <div class="article-info__content">
+          <div class="article-info__title">
+            <h2 >A new way to explore the world</h2>
+          </div>
+          <div class="article-info__text">
```
Comment (suicaed) [inline index.html:52]: There is no need in div wrapper, just use <p> with a class.

File: index.html [hunk 6]
```diff
@@ -41,9 +41,44 @@
...
-    <section class="tt-section"></section>
-    <section class="tt-section"></section>
+
+    <section class="tt-section tt-second-fourth-screen tt-fourthscreen__wrapper">
+      <div class="tt-second-fourth-screen__article-info">
+        <div class="article-info__content">
+          <div class="article-info__title">
+            <h2>Guides by Thousand Sunny</h2>
+          </div>
+          <div class="article-info__text">
```
Comment (suicaed) [inline index.html:71]: There is no need in div wrapper, just use with a class.

File: assets/img/element.svg [hunk 7]
```diff
@@ -0,0 +1,3 @@
+<svg width="17" height="17" viewBox="0 0 17 17" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (suicaed) [inline assets/img/element.svg:1]: What it is used for? Remove unnecessary files.

File: assets/img/element2.svg [hunk 8]
```diff
@@ -0,0 +1,3 @@
+<svg width="21" height="21" viewBox="0 0 21 21" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (suicaed) [inline assets/img/element2.svg:1]: Unnecessary file.

File: assets/img/element3.svg [hunk 9]
```diff
@@ -0,0 +1,9 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (suicaed) [inline assets/img/element3.svg:1]: Unnecessary file.

File: assets/styles/components/secondscreen.css [hunk 10]
```diff
@@ -0,0 +1,42 @@
+.tt-second-fourth-screen__article-info {
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+    flex-direction: row;
+    align-items: center;
+    margin: auto;
```
Comment (suicaed) [inline assets/styles/components/secondscreen.css:7]: margin auto will not work here.

File: assets/styles/components/secondscreen.css [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+}
+.tt-second-fourth-screen__img {
+    width: 50%;
+}
+.article-info__content {
+    width: 40%;
+    display: flex;
```
Comment (suicaed) [inline assets/styles/components/secondscreen.css:14]: Surely we don't need flexbox here.

File: assets/styles/components/secondscreen.css [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+    flex-direction: column;
+    justify-content: space-between;
+}
+.article-info__title {
+    font-size: 36px;
```
Comment (suicaed) [inline assets/styles/components/secondscreen.css:19]: Use relative font size - rem.

**Review by suicaed — CHANGES_REQUESTED** [review]
See comments. Make some changes.
Overall, not bad.
Don't worry, comments is a good stuff. They makes your code better. Code review is a study.

**Comment by ValentinaBrigantina** [comment]
👍

**Review by suicaed — APPROVED** [review]

**Pull request merged**